Gemfile

```diff
@@ -3,6 +3,7 @@ source 'https://rubygems.org'
 gem 'bugsnag-maze-runner', '~>10.0'
 gem 'cocoapods'
 gem 'xcpretty', '~>0.4.0'
+gem 'openssl'
```

Comment:
Was there an issue that let to this being needed?  Something we might need to do in other repos?